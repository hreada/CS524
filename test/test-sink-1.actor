letactor(
  bind(sink, rec(lambda(b, lambda(m, become(b))))),
  seq(send(sink, 0),
  seq(send(sink, 1),
  seq(send(sink, 2),
      send(sink, 3)))))
