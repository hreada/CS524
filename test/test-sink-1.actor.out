<T>
  <actors>
    <actor>
      <k> exec ( nil ) </k>
      <id> _29 </id>
    </actor>
    <actor>
      <k> ready ( lambda ( _5 , app ( app ( lambda ( _4 , app ( lambda ( b , lambda ( m , become ( b ) ) ) , lambda ( _5 , app ( app ( _4 , _4 ) , _5 ) ) ) ) , lambda ( _4 , app ( lambda ( b , lambda ( m , become ( b ) ) ) , lambda ( _5 , app ( app ( _4 , _4 ) , _5 ) ) ) ) ) , _5 ) ) ) </k>
      <id> _1 </id>
    </actor>
    <actor>
      <k> exec ( nil ) </k>
      <id> initactor </id>
    </actor>
    <actor>
      <k> exec ( nil ) </k>
      <id> _10 </id>
    </actor>
    <actor>
      <k> exec ( nil ) </k>
      <id> _17 </id>
    </actor>
    <actor>
      <k> exec ( nil ) </k>
      <id> _24 </id>
    </actor>
  </actors>
  <messages> .MessageCellBag </messages>
  <definedaddr> ListItem ( initactor ) ListItem ( _1 ) ListItem ( _10 ) ListItem ( _17 ) ListItem ( _24 ) ListItem ( _29 ) </definedaddr>
</T>
